### ISSUE TITLE
Could not complete your request because  Brendan's an idiot?

### ISSUE THREAD

**Issue by normhayes** [fnordware/SuperPNG#8]
![brendan s an idiot](https://cloud.githubusercontent.com/assets/307030/25966180/266aac0a-363f-11e7-8f57-6edfb51e0f5e.png)

**Comment by fnordware** [comment]
Haha, yeah, that's an old error thing. Probably really means the PNG file is corrupted somehow. Does it open in other PNG readers?

**Comment by the-real-tokai** [comment]
Happens to me when I want to reload a larger 2700x1800 16-bit PNG file back into Photoshop (one which I just saved via Photoshop).

[edit/ update]

Disabled the SuperPNG plugin and Photoshop itself failed to load the file, so does Affinity Photo. Interestingly Apple's Preview and QuickLook can display the image just fine. Odd stuff.

**Comment by fnordware** [comment]
Hmmmm. I'll see if I can re-create it.

**Comment by fnordware** [comment]
I've been able to write and then read 16-bit PNGs of that size, both with and without Alpha. This is with Photoshop CS6 on Mac. Which version of Photoshop are you using?

**Comment by the-real-tokai** [comment]
@fnordware I don't think this was a SuperPNG problem in my case. Something odd with the file (sadly I can't share that particular one). I'll try to recreate the effect.

One thing that could be improved however is the actual error message. I don't mind the language or anything, but it should include a hint that it's triggered by SuperPNG. It was a real "WTF?" moment there until I looked it up on Google. :)

**Comment by MiroslavRD** [comment]
[r/softwaregore](https://www.reddit.com/r/softwaregore)
